### PR TITLE
deb: add fluent-lts-apt-source package for LTS

### PR DESCRIPTION
**Pull request by kenhys**
We need to maintain 2 channels such as

* Standard version
* LTS version

Each channel must have separate apt-line.

* https://packages.treasuredata.com/5/
* https://packages.treasuredata.com/lts/5

So, for LTS version, provide fluent-lts-apt-source.

### CODE REVIEW

**Comment by daipom** [comment]
We will change here for each branch in the future.
So we don't touch here for now, right?

https://github.com/fluent/fluent-package-builder/blob/a5e12ab9b0846f725923227c5a1babd159b2028a/Rakefile#L22-L24

File: fluent-lts-apt-source/Rakefile [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+require "time"
+
+# Debian
+CLEAN.include("apt/tmp")
+CLEAN.include("apt/build.sh")
+CLEAN.include("apt/env.sh")
+CLEAN.include("debian/tmp")
+CLOBBER.include("apt/repositories")
+
+class FluentdAptSourcePackageTask < PackageTask
```
Comment (daipom) [inline fluent-lts-apt-source/Rakefile:14]: For the same reason, we don't need to separate the class name, right?

Comment (kenhys) [reply fluent-lts-apt-source/Rakefile]: Thanks, I've fixed it.

**Comment by kenhys** [comment]
> We will change here for each branch in the future.
So we don't touch here for now, right?

Yes, it will be fixed in LTS branch.

File: fluent-lts-apt-source/Rakefile (hunk 1 above)
Comment (daipom) [reply fluent-lts-apt-source/Rakefile]: Oh! We needed to fix this! OK!

**Review by daipom — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**